roles/openshift_logging/tasks/install_mux.yaml

```diff
@@ -65,3 +65,10 @@
   failed_when: mux2_output.rc == 1 and 'exists' not in mux2_output.stderr
   check_mode: no
   when: mux_cluster_reader.stdout.find("system:serviceaccount:{{openshift_logging_namespace}}:aggregated-logging-fluentd") == -1
+
+- name: "Add mux namespaces"
```

Comment:
@ewolinetz 
> What does creating multiple namespaces do for mux?

This is only for the common logging case, where logging will be accepting records from multiple different projects.
The use case is described here: https://github.com/openshift/origin-aggregated-logging/blob/master/docs/mux-logging-service.md#tag